build/Makefile

```diff
@@ -40,6 +40,8 @@ VERSION ?= $(base_version)-$(shell git rev-parse --short HEAD)
 REGISTRY ?= $(release_registry)
 # Where the kubectl configuration files are being stored
 KUBEPATH ?= ~/.kube
+# Name of the kubectl configuration file to use
+KUBECONFIGFILE ?= config
```

Comment:
Curious - should we just leverage `KUBECONFIG` rather than the file? So people can overwrite the entire path. Would that be more flexible? (and leverage the same env var that standard Kubernetes tooling uses anyway)

Reply:
I agree the convention is to use `KUBECONFIG` see https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/ default being `$HOME/.kube/config`. We should stick to that so it's more user-friendly and get rid of the KUBEPATH at the same time.

Reply:
ok, will do

Reply:
Done